packages/hardhat-chai-matchers/test/changeTokenBalance.ts

```diff
@@ -150,6 +158,21 @@ describe("INTEGRATION: changeTokenBalance and changeTokenBalances matchers", fun
           );
         });
 
+        it("change balance doesn't satisfies the predicate", async function () {
+          await expect(
+            expect(
+              sender.sendTransaction({ to: receiver.address })
+            ).to.changeTokenBalance(
+              mockToken,
+              sender,
+              (diff: bigint) => diff > BigInt(0)
+            )
+          ).to.be.rejectedWith(
+            AssertionError,
+            /Expected the balance of MCK tokens for "0x\w{40}" to satisfy the predicate, but it didn't \(change by: 0 wei\)/
```

Comment:
For consistency with the `changeEtherBalance` error message, I would say `(token balance change: 0 wei)`